src/studio/src/designer/backend/Controllers/TextController.cs

```diff
@@ -97,27 +98,29 @@ public IActionResult GetLanguages(string org, string app)
         [HttpPost]
         public IActionResult SaveResource([FromBody] dynamic jsonData, string id, string org, string app)
         {
-            id = id.Split('-')[0];
+            // id = id.Split('-')[0];
             JObject json = jsonData;
 
-            JArray resources = json["resources"] as JArray;
-            string[] duplicateKeys = resources.GroupBy(obj => obj["id"]).Where(grp => grp.Count() > 1).Select(grp => grp.Key.ToString()).ToArray();
-            if (duplicateKeys.Length > 0)
-            {
-                return BadRequest($"Text keys must be unique. Please review keys: {string.Join(", ", duplicateKeys)}");
-            }
+            Console.WriteLine(org + app + id);
```

Comment:
This looks like changes in the old controller and perhaps something that shouldn't have been commited?